app/Models/BeatmapDiscussion.php

```diff
@@ -105,7 +105,7 @@ public static function search($rawParams = [])
             $query->withoutDeleted();
         }
 
-        if (!($params['is_moderator'] ?? false)) {
+        if (!($rawParams['is_moderator'] ?? false)) {
```

Comment:
( ¬‿¬)

Reply:
good thing defaults to `false` ( ¬‿¬)

Reply:
can we stick to using sentences to comment on code rather than faces? it's very hard to follow along.